data/campaigns/Eastern_Invasion/lua/wesnoth_lua_pack.lua

```diff
@@ -0,0 +1,251 @@
+
+-- to make code shorter
+local wml_actions = wesnoth.wml_actions
+
+-- metatable for GUI tags
+local T = wml.tag
+
+
+-- from the Wesnoth Lua Pack
+-- [item_dialog]
+-- an alternative interface to pick items
+-- could be used in place of [message] with [option] tags
+function wml_actions.item_dialog( cfg )
```

Comment:
item_dialog_musttake is custom (barely, I just removed a button), but other than that I think it's the same and would be fine to update to the most recent revision, if there's a benefit to doing so.

Note that the filename here is misleading; it's only 2 functions, not the entire lua pack.  I should have renamed it before the PR.

Reply:
The benefit is that current recommended practice is to put the dialog definition in a separate config file instead of inline as Lua code, and the latest WLP has already been updated to do that (at least the latest on GitHub, I don't know if that made it to the add-ons server).